api/responses/members_in_cd.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/*
+    Copyright (C) 2004-2010 Kestas J. Kuliukas / Timothy Jones
+
+	This file is part of webDiplomacy.
+
+    webDiplomacy is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    webDiplomacy is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with webDiplomacy.  If not, see <http://www.gnu.org/licenses/>.
+ */
+namespace webdiplomacy_api;
+
+defined('IN_CODE') or die('This script can not be run by itself.');
+
+require_once ('game_country.php');
+
+/**
+ * Games in CD JSON response
+ * @package webdiplomacy_api
+ */
+class CountriesInCivilDisorder {
+	/**
+	 * List of (gameID, countryID)
+	 * @var array
+	 */
+	public $value = array();
+
+	/**
+	 * Load the countries in CD;
+	 */
+	function load()
+	{
+		global $DB;
+
+		$countryTabl = $DB->sql_tabl("SELECT m.gameID, m.countryID
+                                      FROM wD_Members AS m
+                                      LEFT JOIN wD_Orders AS o ON ( o.gameID = m.gameID AND o.countryID = m.countryID )
+                                      LEFT JOIN wD_Games AS g ON ( g.id = m.gameID )
+                                      WHERE NOT o.id IS NULL
+                                            AND g.variantID in (1, 15, 23)

```

Comment:
Eek, the variant ID can’t really be counted on to be the same from server to server. Can this be added to Config::$something ?

Reply:
Oddly enough, because the variants are made by 1 person, and the variants are then ported to other sites with the variantID defined in the variant install files they're actually identical on all 4 of the webdip code base sites as far as I've been able to tell, so I've unfortunately used this logic in a couple spots. I'll take it upon myself to see if I can come up with a better solution but I'm okay with this in here, and I'll work with squigs44 to get a better design for the few spots we've done this. Sorry about that. 

Definitely agree on the the need for a config setting to be able to entirely turn off the api/ensure a site without it setup won't have problems though.